apps/cli-gui/src/app/app.component.ts

```diff
@@ -7,7 +7,7 @@ import { CoreService } from './core/core.service';
 
 @Component({
   standalone: true,
-  imports: [NgFor, NgIf, AsyncPipe, RouterOutlet, MatSelectModule],
+  imports: [NgFor, NgIf, AsyncPipe, RouterOutlet, MatSelectModule, NgForOf],
```

Comment:
validate if you need both NgFor and NgForOf

Reply:
done